charts/aws-calico/values.yaml

```diff
@@ -0,0 +1,57 @@
+fullnameOverride: calico
+
+serviceAccount:
+  create: true
+
+podSecurityPolicy:
+  create: false
+
+calico:
+  tag: v3.15.1
+
+  typha:
+    logseverity: Info #Debug, Info, Warning, Error, Fatal
+    image: quay.io/calico/typha
+    resources:
+      requests:
+        memory: "64Mi"
+        cpu: "50m"
+      limits:
+        memory: "96Mi"
+        cpu: "100m"
+    tolerations: []
+    nodeSelector:
+      beta.kubernetes.io/os: linux
```

Comment:
okay, I see the OS node selector here 👍   but still the question on arm64 arch support. 

Also, the beta.kubernetes.io node selector is deprecated. Looks like you have it properly handled in the vpc-cni chart

Reply:
Right, will fix this.